iOSClient/Login/NCLoginWeb.swift

```diff
@@ -123,15 +123,24 @@ class NCLoginWeb: UIViewController {
             let error = NKError(errorCode: NCGlobal.shared.errorInternalError, errorDescription: "_login_url_error_")
             NCContentPresenter.shared.showError(error: error, priority: .max)
         }
-
-        // TITLE
-        if let host = URL(string: urlBase)?.host {
-            titleView = host
-            if let account = NCManageDatabase.shared.getActiveAccount(), CCUtility.getPassword(account.account).isEmpty {
-                titleView = NSLocalizedString("_user_", comment: "") + " " + account.userId + " " + NSLocalizedString("_in_", comment: "") + " " + host
-            }
+        
```

Comment:
I guess this customization can be extracted to another PR if there are no dependency with onboarding customization.